features/step-definitions/compose.steps.js

```diff
@@ -0,0 +1,25 @@
+const { Given, When, Then } = require('@wdio/cucumber-framework');
+
+const composePage = require('../pageobjects/compose.page');
+
+Then(/^I should see a Compose page with Address, Subject and Body fields$/, async () => {
+  await expect(composePage.inputAddressField).toExist();
+  await expect(composePage.inputSubjectField).toExist();
+  await expect(composePage.inputBodyField).toExist();
+});
+  
+When(/^I fill the Address field with 'test1234512345@yopmail.com'$/, async () => {

```

Comment:
do not use hardcoded value in steps to reuse the same steps for different tests

When(/^I fill the Address field with 'test1234512345@yopmail.com'$/, async (emailAddress) => {
  await composePage.inputAddressField.setValue(emailAddress);
});